lib/route.js

```diff
@@ -322,7 +322,7 @@ function buildRouting (options) {
         constraints.version = opts.version
       }
 
-      const headHandler = router.find('HEAD', opts.url, constraints)
+      const headHandler = router.findRoute('HEAD', opts.url, constraints)
       const hasHEADHandler = headHandler != null
```

Comment:
nit: if findRoute can’t return undefined, can you make this a strict check

```suggestion
      const hasHEADHandler = headHandler !== null
```